components/DescriptionList/src/DescriptionListDetail.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { HTMLAttributes } from 'react';
+
+export type DescriptionListDetailProps = HTMLAttributes<HTMLElement>;
+
+export const DescriptionListDetail: React.FC<DescriptionListDetailProps> = ({
+  children,
+}: DescriptionListDetailProps) => {
+  return <dd className="denhaag-description-list__detail">{children}</dd>;
```

Comment:
You should pass the other props to the dd tag, like you did in the DescriptionListCaption.tsx, so it's more flexible in case a developer doesn't want to use the smart DescriptionList component, but wants to use the atomic components.